.github/dependabot-auto-merge.yml

```
@@ -0,0 +1,12 @@
+name: Dependabot Auto Merge
+on:
+  pull_request:
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: minor
+          github-token: ${{ secrets.DependabotAutoMerge }}
```

The naming convention for secrets is UPPER_SLUG_CASE, and I think it should be explicit that this is a token.

```suggestion
          github-token: ${{ secrets.DEPENDABOT_AUTO_MERGE_TOKEN }}
```